sw/airborne/modules/actuators/actuators.h

```diff
@@ -72,7 +79,7 @@ extern int16_t actuators_pprz[ACTUATORS_NB];
  * @param _n actuators name as given in airframe file, servos section
  * @param _v new actuator's value
  */
-#define _ActuatorSet(_n, _v) Set_ ## _n ## _Servo(_v)
+#define _ActuatorSet(_n, _v) Set_ ## _n ## _Servo(_v,_v)
```

Comment:
Here you pass the value `_v` twice, but they should be pprz and driver values. It seems that something is wrong. We either do the mapping pprz to driver should be done in `Set_XXX_Servo` functions (and they take pprz inputs), or we store the `pprz_val` before calling this function (and they take driver inputs as it is currently the case).

Reply:
This macro is weird indeed and points to an inconsistency in the project: it is used in code that sets actuator values directly: e.g. switch servo sets the actuator to 1200 us. The correct way would be to put it to MAX_PPRZ. I can harmonize that.

Reply:
I'm in favor of harmonizing the servo setter with PPRZ values. It would make the code actuator independent.

Reply:
It was now moved to the ```Set_XXX_Servo``` function. Now all pan-tilt and switch servo code becomes much simpler.